avm/res/web/site/tests/e2e/logsConfig/dependencies.bicep

```
@@ -0,0 +1,36 @@
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Application Insights instance to create.')
+param applicationInsightsName string
+
+@description('Required. The name of the Server Farm to create.')
+param serverFarmName string
+
+resource serverFarm 'Microsoft.Web/serverfarms@2022-03-01' = {
+  name: serverFarmName
+  location: location
+  sku: {
+    name: 'S1'
+    tier: 'Standard'
+    size: 'S1'
+    family: 'S'
+    capacity: 1
+  }
+  properties: {}
+}
+
+resource applicationInsights 'Microsoft.Insights/components@2020-02-02' = {
+  name: applicationInsightsName
+  location: location
+  kind: ''
+  properties: {
+    Application_Type: 'web'
+  }
+}
+
+@description('The resource ID of the created application insights.')
+output applicationInsigtsId string = applicationInsights.id
```

```suggestion
output applicationInsigtsResourceId string = applicationInsights.id
```

Recommend to always use the full name to avoid confusion with other types of ids